src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureQuery.java

```diff
@@ -107,27 +122,50 @@ BanyandbMeasure.QueryRequest build() throws BanyanDBException {
         builder.setMetadata(buildMetadata());
         if (timestampRange != null) {
             builder.setTimeRange(timestampRange.build());
+        } else {
+            builder.setTimeRange(TimestampRange.MAX_RANGE);
```

Comment:
@hanahmily Should we consider this as a temp fix?

Reply:
Or, should we report an error in measure?

Reply:
I'm amending the server side with a default time range. Could we hold this PR for a while?

Reply:
Sure, we could wait for your side work.